R/derive_date_vars.R

```diff
@@ -440,12 +961,12 @@ convert_dtc_to_dtm <- function(dtc,
 #'
 #' @inheritParams convert_dtc_to_dtm
 #'
-#' @author Samia Kabi
-#'
 #' @details Usually this computation function can not be used with `%>%`.
 #'
 #' @return A datetime object
 #'
+#' @author Samia Kabi
+#'
 #' @keywords computation timing
 #'
```

Comment:
Will need to update keywords and families

Reply:
Updated